src/firmware/app/control/control.h

```diff
@@ -5,7 +5,7 @@
 /**
  * Apply the given acceleration (in robot coordinates) to the given robot
  *
- * @param robot The robot to apply acceleration to
+ * @param robot [in] The robot to apply acceleration to
```

Comment:
this is an exception because the first argument is a class

Reply:
Done.